python_modules/dagster-graphql/dagster_graphql/schema/logs/events.py

```diff
@@ -299,6 +299,15 @@ class Meta:
         name = "HookErroredEvent"
 
 
+class GrapheneLogManagerMetadata(graphene.ObjectType):
+    class Meta:
+        name = "LogManagerMetadata"
+
+    logManagerClass = graphene.String(required=True)
+    storageAccount = graphene.String()
+    container = graphene.String()
```

Comment:
i am not sure what `container` means as someone without context - consider more specific name